explainer.md

```diff
@@ -180,70 +180,73 @@ Media Capabilities should offer a means of surfacing when different MediaDecodin
+For HDR support detection, there are three main components whose capabilities need to be surfaced -- the decoder, renderer, and display. The decoder takes in an encoded stream and produces a decoded stream understood by the renderer, which in turn maps the stream's signals to those the display can properly output. Most of the time, the decoder and renderer are part of the UA while the display intuitively describes the physical output monitor, whether this be a computer monitor or TV. To match this natural modularity between the UA and the display, this API is compartmentalized into two (2) parts:
 
-#### Colour/Pixel Depth
+*   *MediaCapabilities.decodingInfo()*: handles the UA pieces, namely decoding and rendering. 
+*   TODO: various aspects of the display are being discussed. 
 
-It is already exposed on the *Screen* object but only for compatibility reasons. The [CSSOM View Module](https://www.w3.org/TR/cssom-view-1/#dom-screen-pixeldepth) should be updated or amended to make this information available.
-
-#### Brightness
-
-The minimum and maximum brightness should be exposed on the *Screen* object. In order to know the effective brightness, a website would need to know the brightness of the room which can be achieved with the [Ambient Light Sensor](https://w3c.github.io/ambient-light/).
-
-#### Example
+### Screen capabilities
 
-```JavaScript
-function canDisplayMyHDRStreams() {
-  // The conditions below are entirely made up :)
-  return window.screen.colorGamut == "rec2020" &&
-         window.screen.pixelDepth == "30" &&
-         window.screen.brightness.max > 500 &&
-         window.screen.brightness.min < 0.1;
-}
-```
+Even if a device is able to decode HDR content, if the screen isn’t able to show this content appropriately, it might not be worth using HDR content for the website because of the higher bandwidth consumptions but also the rendering might be worse than a SDR optimised video.
 
-### Screen change
+The shape of this API is actively being discussed, specifically how TVs' two-plane problem should be handled. Please refer to issue [#135](https://github.com/w3c/media-capabilities/issues/135).
 
-The Web Platform only exposes the current screen associated to the website window. That means that a window changing screen will get its `window.screen` updated. A page can poll to find out about this but adding a *change* event to the *Screen* interface might be a more efficient way to expose this information.
+***Work in progress***
 
 ### Decode capabilities
 
 HDR videos have some information that need to be understood by the user agent in order to be rendered correctly. A website might want to check that the user agent will be able to interpret its HDR content before providing it.
```

Comment:
Suggest deleting these two sentences, they are redundant with earlier sections and the following text.